focus-areas/what/HardwareDiversity.txt

```diff
@@ -0,0 +1,37 @@
+﻿Hardware Diversity

```

Comment:
We think you intend to talk about Hardware as a barrier to joining a project.  Could you make that more clear.